src/rpc/rawtransaction.cpp

```diff
@@ -873,15 +873,16 @@ UniValue signrawtransaction(const JSONRPCRequest& request)
 
 UniValue sendrawtransaction(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.size() < 1 || request.params.size() > 3)
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 4)
         throw std::runtime_error(
-            "sendrawtransaction \"hexstring\" ( allowhighfees instantsend )\n"
+            "sendrawtransaction \"hexstring\" ( allowhighfees instantsend bypasslimits)\n"
             "\nSubmits raw transaction (serialized, hex-encoded) to local node and network.\n"
             "\nAlso see createrawtransaction and signrawtransaction calls.\n"
             "\nArguments:\n"
             "1. \"hexstring\"    (string, required) The hex string of the raw transaction)\n"
             "2. allowhighfees  (boolean, optional, default=false) Allow high fees\n"
             "3. instantsend    (boolean, optional, default=false) Use InstantSend to send this transaction\n"
+            "3. bypasslimits   (boolean, optional, default=false) Bypass transactions policy limits\n"
```

Comment:
4*

Reply:
Sigh... Thanks, will include in some future "cleanup PR" 👍